src/components/conjunction/ConjunctionMissDistanceChart.tsx

```diff
@@ -1,26 +1,27 @@
 import Link from 'next/link';
-import { useTranslations } from 'next-intl';
+import { getTranslations } from 'next-intl/server';
 
+import type { TypeEventSummaryOut } from '@/__generated__/data-contracts';
 import Details from '@/ui/details/details';
 
-import RichText from '../RichText';
+import MissDistanceChart from '../charts/miss-distance-chart/MissDistanceChart';
 
 type ConjunctionMissDistanceChartProps = {
-  id: string;
+  shortId: string;
+  events: TypeEventSummaryOut[];
+  isSpecial: boolean;
 };
 
-const ConjunctionMissDistanceChart = ({ id }: ConjunctionMissDistanceChartProps) => {
-  const t = useTranslations('Accordions.Conjunction.miss_distance_chart');
+const ConjunctionMissDistanceChart = async ({ shortId, events, isSpecial }: ConjunctionMissDistanceChartProps) => {
+  const t = await getTranslations('Accordions.Conjunction.miss_distance_chart');
```

Comment:
Same, this should not be in Accordion translations

Reply:
Fixed